charts/jenkins/jobs/opensearch2/nuxeo-opensearch2.groovy.gotmpl

```diff
@@ -0,0 +1,41 @@
+multibranchPipelineJob('opensearch2/nuxeo-opensearch2') {
+  // Description
+  description('Nuxeo OpenSearch2')
+  // Branch Sources
+  branchSources {
+    branchSource {
+      source {
+        // GitHub
+        github {
+          id('opensearch2/nuxeo-opensearch2')
+          // Credentials
+          credentialsId('github-branch-source')
+          // Repository
+          repoOwner('nuxeo')
+          repository('nuxeo-opensearch2')
+          repositoryUrl('https://github.com/nuxeo/nuxeo-opensearch2')
+          configuredByUrl(false)
+          // Behaviors
+          traits {
+            {{- .Values.jobs.traits.gitHubBranchDiscovery | nindent 12 }}
+            {{- .Values.jobs.traits.gitHubPullRequestDiscovery | nindent 12 }}
+            {{- .Values.jobs.traits.disableStatusUpdateTrait | nindent 12 }}
+            {{- .Values.jobs.traits.gitHubStatusChecks | nindent 12 }}
+            {{- .Values.jobs.traits.packageLtsHeadRegexFilter | nindent 12 }}
```

Comment:
This (aka branches will be lts-2023, lts-2025), or `ltsHeadRegexFilter`, for 2023, 2025 branches.
In fact, don't we only want 2025?